### ISSUE TITLE
Field alias does not maintain case

### ISSUE THREAD

**Issue by denniswambua** [ambitioninc/django-query-builder#54]
Created a field with camel casing 
` CountField(field="json->>'%s'" % unicode(column), alias="Count")]`
The results comes with the count as all small letters.

`[{'count': 7L, 'gender': u'male'}, {'count': 1L, 'gender': u'female'}]`

Is there a way to maintain the casing as set in the alias property?

**Comment by denniswambua** [comment]
This is a postgres thing, i got it working by passing alias with double quotes. 
`'"Count"'`
Full example
`CountField(field="json->>'%s'" % unicode(column), alias='"{}"'.format('Count'))]`

Is there a better way to achieve this?

**Comment by wesokes** [comment]
@denniswambua You are right, postgres will lower the case unless the alias is in double quotes. I'll update the project to automatically add double quotes around the alias so you don't have to manually do this.

**Comment by denniswambua** [comment]
awesome that would be great
